src/test/resources/emails/sessionClosedEmailForInstructor.html

```diff
@@ -0,0 +1,13 @@
+Hello Instructor1 Course1,
+
+This is a gentle reminder sent to instructors to let you know that the following session is now closed.
```

Comment:
If we send the email to everyone, the text can be changed to,
`The following session is now closed. You can still view your submission by going to the link sent earlier, but you will not be able to edit existing responses or submit new responses.`
... session details here
`Any request for deadline extensions should be sent to the instructors of the course, not to the TEAMMATES support team. If you need any help or clarifications regarding TEAMMATES ...`